Jellyfin.Server/SocketSharp/RequestMono.cs

```diff
@@ -195,10 +211,15 @@ public void ValidateInput()
 
         bool IsContentType(string ct, bool starts_with)
         {
-            if (ct == null || ContentType == null) return false;
```

Comment:
Not sure I like this change, one-liner was just as readable. If this squashes a warning, though, let's do it.